docs/guides/custom-schema.md

```diff
@@ -197,14 +197,16 @@ Add the query alongside our custom type and mutation. Note that custom queries t
 And finally, define our query's resolver:
 
 ```javascript
-const pageViewsOver = async (_, { id, threshold }, _, _, { query }) => {
+const pageViewsOver = async (_, { id, threshold }, context) => {
```

Comment:
I know this doesn't really have much to do with this PR, but I find these unnamed variables are a bit confusing to read.

Reply:
I think of them as placeholders only. :)